code/modules/antagonists/eldritch_cult/eldritch_items.dm

```diff
@@ -0,0 +1,118 @@
+/obj/item/living_heart
+	name = "Living Heart"
+	desc = "Link to the worlds beyond."
+	icon = 'icons/obj/eldritch.dmi'
+	icon_state = "living_heart"
+	w_class = WEIGHT_CLASS_SMALL
+	///Target
+	var/mob/living/carbon/human/target
+
+/obj/item/living_heart/attack_self(mob/user)
+	. = ..()
+	if(!IS_E_CULTIST(user))
+		return
+	if(!target)
+		to_chat(user,"<span class='warning'>No target could be found. Put the living heart on the rune and use the rune to recieve a target.</span>")
+		return
+	var/dist = get_dist(user.loc,target.loc)
+	var/dir = get_dir(user.loc,target.loc)
+
+	switch(dist)
+		if(0 to 15)
+			to_chat(user,"<span class='warning'>[target.real_name] is near you. They are to the [lowertext("[dir]")] of you!</span>")
+		if(16 to 31)
+			to_chat(user,"<span class='warning'>[target.real_name] is somewhere in your vicinty. They are to the [lowertext("[dir]")] of you!</span>")
+		if(32 to 127)
+			to_chat(user,"<span class='warning'>[target.real_name] is far away from you. They are to the [lowertext("[dir]")] of you!</span>")
+		else
+			to_chat(user,"<span class='warning'>[target.real_name] is beyond our reach.</span>")
+
+	if(target.stat == DEAD)
+		to_chat(user,"<span class='warning'>[target.real_name] is dead. Bring them onto a transmutation rune!</span>")
+
+/obj/item/melee/sickly_blade
+	name = "Sickly blade"
+	desc = "Twisted sickle with an ornamental eye. The eyes looks at you."
+	icon = 'icons/obj/eldritch.dmi'
+	icon_state = "eldritch_blade"
+	inhand_icon_state = "eldritch_blade"
+	lefthand_file = 'icons/mob/inhands/weapons/swords_lefthand.dmi'
```

Comment:
Double check your inhands, I suspect the lefthand ones are backwards.

Reply:
good catch